docs/en/pid-eaa-issuance.rst

```diff
@@ -439,50 +454,152 @@ If the checks defined above are successful the Wallet Instance proceeds with the
   :language: JSON 
 
 
-**Step 21 (Notification Response)**: When the Credential Issuer has successfully received the Notification Request from the Wallet, it MUST respond with an HTTP status code *204* as recommended in Section 10.2 of [`OpenID4VCI`_]. Below is a non-normative example of response to a successful Notification Request:
+**Step 23 (Notification Response)**: When the Credential Issuer has successfully received the Notification Request from the Wallet, it MUST respond with an HTTP status code *204* as recommended in Section 10.2 of [`OpenID4VCI`_]. Below is a non-normative example of response to a successful Notification Request:
 
 .. code-block:: http
 
   HTTP/1.1 204 No Content
 
 
+Refresh Token Flow
+------------------
 
-Deferred Flow
--------------
+To use the Deferred endpoint, Credential Request endpoint and the Notification endpoint the Wallet Instance MUST present to the Credential Issuer a valid DPoP Access Token. However, when these endpoints are used for the deferred flow, to re-issue a Digital Credential or to notify the deletion of a Digital Credential respectively, they may lead to the Access Token expiration as the validity of the Access Token MUST be short in time and these processes may happen several days after the release of the Access Token. To enable these scenarios, this specification requires the use of Refresh Tokens.
```

Comment:
To use the the Deferred, Credential Request, and Notification endpoints, the Wallet Instance MUST present a valid DPoP Access Token to the Credential Issuer. However, when these endpoints are used in the Deferred Flow, for re-issuing or notifying the deletion of a Digital Credential, the Access Token might expire, as it is designed to be short-lived and these actions MAY occur days later. To address this, the specification RECOMMENDS the use of Refresh Tokens.